velox/exec/GroupingSet.cpp

```diff
@@ -364,10 +364,22 @@ bool GroupingSet::getGlobalAggregationOutput(
       aggregates_[i]->extractValues(groups, 1, &result->childAt(i));
     }
   }
+
+  destroyGlobalAggregations();
```

Comment:
Shall we call this in GroupingSet dtor? Thanks!

Reply:
@ebeworld Can you move this to dtor to be aligned with `StreamingAggregation`?

Reply:
Done.